### PR TITLE
Added no devices state

### PR DESCRIPTION
**Pull request by ryanfeeley**
Also fixed typos

### CODE REVIEW

File: features/FxA-16-devices-view/README.md [hunk 1]
```diff
@@ -34,19 +34,23 @@ UI Mocks
 
 ## Collapsed View
 When the user is syncing one or more devices, a **Devices** section appears in the settings section above the **Password** section.
```
Comment (vladikoff) [inline features/FxA-16-devices-view/README.md]: Should we get rid of this line? or change it?

Comment (ryanfeeley) [inline features/FxA-16-devices-view/README.md]: Removed "when…" and committed and synced.

**Comment by chrismore** [comment]
Let's use this URL for all devices. It will work for all OS and all languages. Make sure it has these UTM parameters and that it doesn't contain the locale in the URL. It will redirect to the right language and also provide a button for desktop, android and iOS.

https://www.mozilla.org/firefox/new/?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices

**Comment by vladikoff** [comment]
@chrismore Thank you!

**Comment by chrismore** [comment]
Welcome!

**Comment by vladikoff** [comment]
Thanks @ryanfeeley !!

**Pull request merged**